### PR TITLE
Update template based on latest version

### PR DESCRIPTION
**Pull request by man4toman**

### CODE REVIEW

File: templates/wp-config.mustache [hunk 1]
```diff
@@ -75,8 +75,9 @@ define( 'WPLANG', '{{locale}}' );
 /* That's all, stop editing! Happy publishing. */
 
 /** Absolute path to the WordPress directory. */
-if ( ! defined( 'ABSPATH' ) )
+if ( ! defined( 'ABSPATH' ) ){
```
Comment (schlessera) [inline templates/wp-config.mustache]:
```suggestion
if ( ! defined( 'ABSPATH' ) ) {
```

File: templates/wp-config.mustache [hunk 2]
```diff
@@ -75,8 +75,9 @@ define( 'WPLANG', '{{locale}}' );
...
 	define( 'ABSPATH', dirname( __FILE__ ) . '/' );
+}
 
 /** Sets up WordPress vars and included files. */
-require_once ABSPATH . 'wp-settings.php';
+require_once( ABSPATH . 'wp-settings.php' );
```
Comment (schlessera) [inline templates/wp-config.mustache]: Is that nonsense still a thing in Core? I thought everything ran through PHPCS... ?

Comment (man4toman) [reply templates/wp-config.mustache]: Yes, it still there :)
See here: https://github.com/WordPress/WordPress/blob/master/wp-config-sample.php#L90

Comment (man4toman) [reply templates/wp-config.mustache]: Maybe it needs a ticket to fix!

Comment (schlessera) [reply templates/wp-config.mustache]: Can you remove the parentheses again? I don't think I want WP-CLI to further encourage this and add to the confusion.

File: templates/wp-config.mustache [hunk 3]
```diff
@@ -75,8 +75,9 @@ define( 'WPLANG', '{{locale}}' );
...
 
 /** Absolute path to the WordPress directory. */
-if ( ! defined( 'ABSPATH' ) )
+if ( ! defined( 'ABSPATH' ) ) {
 	define( 'ABSPATH', dirname( __FILE__ ) . '/' );
+}
 
 /** Sets up WordPress vars and included files. */
-require_once ABSPATH . 'wp-settings.php';
+require_once( ABSPATH . 'wp-settings.php' );
```
Comment (schlessera) [inline templates/wp-config.mustache]:
```suggestion
require_once ABSPATH . 'wp-settings.php';
```

**Comment by schlessera** [comment]
Thanks for the PR, @man4toman !

**Pull request merged**